lib/_http_common.js

```diff
@@ -52,9 +52,9 @@ const MAX_HEADER_PAIRS = 2000;
 function parserOnHeaders(headers, url) {
   // Once we exceeded headers limit - stop collecting them
   if (this.maxHeaderPairs <= 0 ||
-      this._headers.length < this.maxHeaderPairs) {
```

Comment:
Style changes like this are unnecessary, besides, I actually prefer having braces when either the conditional or the body take up multiple lines, for easier readability.